inmem_signal_test.go

```diff
@@ -31,7 +31,10 @@ func TestInmemSignal(t *testing.T) {
 	time.Sleep(15 * time.Millisecond)
 
 	// Send signal!
-	syscall.Kill(os.Getpid(), syscall.SIGUSR1)
+	err := syscall.Kill(os.Getpid(), syscall.SIGUSR1)
+	if err != nil {
+		t.Fatalf("failed to signal process: %s", err)
+	}
```

Comment:
What's the circumstance that would hit this error?

Reply:
That would be platform dependent. After reading the kill(2) man page for MacOS and Linux, I don't see any obvious way for this syscall to fail. Nevertheless, I believe the test won't negatively impact the test.